src/client/components/pages/parts/admin-panel-search-results.tsx

```diff
@@ -0,0 +1,160 @@
+import {genEntityIconHTMLElement} from '../../../helpers/entity';
+import {getPrivilegeShieldIcon} from '../../../../common/helpers/privileges-utils';
+
+
+const {Button, Table} = bootstrap;
+
+type AdminPanelSearchResultsState = {
+	selectedUser?: Record<string, unknown>,
+	showModal: boolean
+};
+type AdminPanelSearchResultsProps = {
+	results?: any[],
+	updateResultsOnPrivsChange: () => void,
+	user: Record<string, unknown>
+};
+
+
+/**
+ * Renders the document and displays the 'AdminPanelSearchResults' page.
+ * @returns {ReactElement} a HTML document which displays the AdminPanelSearchResults.
+ * @param {object} props - Properties passed to the component.
+ */
+class AdminPanelSearchResults extends React.Component<AdminPanelSearchResultsProps, AdminPanelSearchResultsState> {
+	static displayName = 'AdminPanelSearchResults';
+
+	static propTypes = {
+		results: PropTypes.array,
+		updateResultsOnPrivsChange: PropTypes.func.isRequired,
+		user: PropTypes.object.isRequired
+	};
+
+	static defaultProps = {
+		results: null
+	};
+
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			selectedUser: null,
+			showModal: false
+		};
+		this.onCloseModal = this.onCloseModal.bind(this);
+	}
+
+	onCloseModal() {
+		this.setState({showModal: false});
+	}
+
+	openPrivsEditModal(user) {
+		this.setState({
+			selectedUser: user,
+			showModal: true
+		});
+	}
+
+	render() {
+		const noResults = !this.props.results || this.props.results.length === 0;
+		const results = this.props.results.map((result) => {
+			if (!result) {
+				return null;
+			}
+			const name = result.defaultAlias ? result.defaultAlias.name :
+				'(unnamed)';
+			const link = `/editor/${result.bbid}`;
+
+			/* eslint-disable react/jsx-no-bind */
+			return (
+				<tr key={result.bbid}>
+					<td>
+						<a href={link}>
+							{genEntityIconHTMLElement('editor')}
+							{name}
+						</a>
+					</td>
+					<td>
+						<PrivilegeBadges privs={result.privs}/>
+					</td>
+					<td>
+						<Button
+							variant="link"
+							onClick={() => this.openPrivsEditModal(result)}
+						>
+							<img height="18" src={getPrivilegeShieldIcon(result.privs)}/> {' '}
+							<FontAwesomeIcon icon={faPencilAlt}/>
+						</Button>
+					</td>
+				</tr>
+			);
+		});
+		const tableCssClasses = 'table table-striped';
+		if (noResults) {
+			return null;
+		}
+		return (
+			<div>
+				{
+					this.state.showModal &&
+					(
+						<div>
+							<PrivsEditModal
+								adminId={this.props.user.id}
+								handleCloseModal={this.onCloseModal}
+								show={this.state.showModal}
+								targetUser={this.state.selectedUser}
+								updateResultsOnPrivsChange={this.props.updateResultsOnPrivsChange}
+							/>
+						</div>
+					)
+				}
+				<h3 style={{color: '#754e37'}}>
+					Search Results
+				</h3>
+				<hr className="thin"/>
+				<Table
+					responsive
+					className={tableCssClasses}
+				>
+					<thead>
+						<tr>
+							<th className="col-md-5">Name</th>
```

Comment:
Using bootstrap class names here does not work on Safari (we just fixed that same issue across the codebase in #987, it's fresh on my mind :) ).

You can use the `width` property (native to the `<th>` element) to the same effect. The value should be `width="42%"` which is what col-md-5 does, although I have found that the following values work well:

![image](https://github.com/metabrainz/bookbrainz-site/assets/6179856/9f1dcf10-f541-428a-9a16-0b1f54a55baa)